src/main/java/sage/tasktypes/Task.java

```diff
@@ -0,0 +1,29 @@
+package sage.tasktypes;
+
+public class Task {
+    private String taskName = "";
+    private boolean completed = false;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.completed = false;
+    }
+
+    public String getTaskName() {
+        return this.taskName;
```

Comment:
```suggestion
        return taskName;
```
Try to standardize what you use throughout your code